www/nginx/src/opnsense/www/js/nginx/src/templates/logviewer.html

```diff
@@ -31,46 +31,46 @@
+            <th class="active"><input type="text" value="<%= model.escape('size') %>" name="size" /></th>
+            <th class="active"><input type="text" value="<%= model.escape('http_referer') %>" name="http_referer" /></th>
+            <th class="active"><input type="text" value="<%= model.escape('user_agent') %>" name="user_agent" /></th>
+            <th class="active"><input type="text" value="<%= model.escape('forwarded_for') %>" name="forwarded_for" /></th>
+            <th class="active"><input type="text" value="<%= model.escape('request_line') %>" name="request_line" /></th>
             <% } else { %>
-            <th><input type="text" value="<%= model.escape('time') %>" name="time" /></th>
-            <th><input type="text" value="<%= model.escape('remote_ip') %>" name="remote_ip" /></th>
-            <th><input type="text" value="<%= model.escape('status') %>" name="status" /></th>
-            <th><input type="text" value="<%= model.escape('bytes_sent') %>" name="bytes_sent" /></th>
-            <th><input type="text" value="<%= model.escape('bytes_received') %>" name="bytes_received" /></th>
-            <th><input type="text" value="<%= model.escape('session_time') %>" name="session_time" /></th>
+            <th class="active"><input type="text" value="<%= model.escape('time') %>" name="time" /></th>
+            <th class="active"><input type="text" value="<%= model.escape('remote_ip') %>" name="remote_ip" /></th>
+            <th class="active"><input type="text" value="<%= model.escape('status') %>" name="status" /></th>
+            <th class="active"><input type="text" value="<%= model.escape('bytes_sent') %>" name="bytes_sent" /></th>
+            <th class="active"><input type="text" value="<%= model.escape('bytes_received') %>" name="bytes_received" /></th>
+            <th class="active"><input type="text" value="<%= model.escape('session_time') %>" name="session_time" /></th>
             <% } %>
         </tr>
     </thead>
     <tbody>
     </tbody>
     <tfoot class="sticky-bottom">
         <tr>
-            <th class="text-nowrap">
+            <th class="text-nowrap active">
                 <button class="btn btn-primary" id="paging_first" title="First Page"><i class="fa fa-fast-backward" aria-hidden="true"></i></button>
                 <button class="btn btn-primary" id="paging_back" title="Previous Page"><i class="fa fa-step-backward" aria-hidden="true"></i></button>
                 <button class="btn btn-primary" id="refresh" title="Refresh"><i class="fa fa-refresh" aria-hidden="true"></i></button>
                 <button class="btn btn-primary" id="paging_forward" title="Next Page"><i class="fa fa-step-forward" aria-hidden="true"></i></button>
                 <button class="btn btn-primary" id="paging_last" title="Last Page"><i class="fa fa-fast-forward " aria-hidden="true"></i></button>
             </th>
-            <th>
+            <th class="active">
```

Comment:
why do you need the active class here?

Reply:
I agree, this looks strange, but it's the only way I found to highlight a header row which is not the first line without modifying all themes.
As here we have two header lines (two `tr`-elements within the `thead`) the current CSS selectors do not match:
```css
.table > caption + thead > tr:first-child > th, .table > caption + thead > tr:first-child > td, .table > colgroup + thead > tr:first-child > th, .table > colgroup + thead > tr:first-child > td, .table > thead:first-child > tr:first-child > th, .table > thead:first-child > tr:first-child > td {
    font-family: 'SourceSansProSemibold';
    font-weight: normal;
    color: #FFF;
}
```

(This is equal for the OPNsense default theme as well as e.g. for the cicada theme.)

Reply:
If it is too much effort do use a "correct" class, then keep it as it is.